schedule_lessons/dashboard/static/dashboard/js/agenda.js

```diff
@@ -1,54 +1,58 @@
-$(document).ready(function () {
+$(document).ready(function() {
   $('[data-toggle="tooltip"]').tooltip();
 
-  $(".acknowledge-btn").on('click', function(e) {
+  $('.acknowledge-btn').on('click', function(e) {
     let data = {
-      'show_tutorial': false
+      show_tutorial: false
     };
-    if (!$('#dontShowTutorialCheckbox').is(":checked")) {
+    if (!$('#dontShowTutorialCheckbox').is(':checked')) {
       data.show_tutorial = true;
     }
-    $(".tutorial-wrapper").remove();
+    $('.tutorial-wrapper').remove();
     $.ajax({
       url: '/dashboard/save_tutorial_preferences/',
       type: 'post',
       data: data,
       error: function(xhr, status) {
-        alert("Something went wrong, please try again.");
+        alert('Something went wrong, please try again.');
       }
     });
   });
 
-  $('#cancelLessonModal').on('show.bs.modal', function (e) {
-    document.getElementById('cancelLessonConfirm').setAttribute("href", e.relatedTarget.href);
+  $('#cancelLessonModal').on('show.bs.modal', function(e) {
+    document
+      .getElementById('cancelLessonConfirm')
+      .setAttribute('href', e.relatedTarget.href);
   });
 
-  $('#options_btn').click(function(){
-    $('.two-buttons').fadeToggle(300).toggleClass('addAnim');
+  $('#options_btn').click(function() {
+    $('.two-buttons')
+      .fadeToggle(300)
+      .toggleClass('addAnim');
     $('#options_btn').toggleClass('addBtnAnim');
   });
 
-  $('#secondNavBtn').click(function(){
+  $('#secondNavBtn').click(function() {
     $('#navbarSupportedContent').slideToggle('fast');
   });
 
-  $('#declineLessonModal').on('show.bs.modal', function (e) {
-    document.getElementById('declineLessonConfirm').setAttribute("href", e.relatedTarget.href);
+  $('#declineLessonModal').on('show.bs.modal', function(e) {
+    document
+      .getElementById('declineLessonConfirm')
+      .setAttribute('href', e.relatedTarget.href);
   });
 
-  $('#notificationsDropdownLink').click(function () {
+  $('#notificationsDropdownLink').click(function() {
     var attr = $('#notificationIcon').attr('data-count');
     if (typeof attr !== typeof undefined && attr !== false) {
```

Comment:
what is this check trying to assert? I feel like it can be simplified to even just:
`if (attr)`